### PR TITLE
Update EMR instructions

### PR DESCRIPTION
**Pull request by ns1123**

### CODE REVIEW

File: docs/en/compute/AWS-EMR.md [hunk 1]
```diff
@@ -44,8 +44,8 @@ To begin with, [download an Alluxio release](https://www.alluxio.io/download) an
 $ aws emr create-default-roles
 ```
 2. The Alluxio bootstrap script is hosted in a publicly readable
-[S3 bucket](https://alluxio-public.s3.amazonaws.com/emr/2.0.1/alluxio-emr.sh).
-This bucket can also be accessed using it's S3 URI: `s3://alluxio-public/emr/2.0.1/alluxio-emr.sh`
+[S3 bucket](https://alluxio-public.s3.amazonaws.com/emr/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-emr.sh).
```
Comment (ZacBlanco) [inline docs/en/compute/AWS-EMR.md]: curious is this will always be true...

File: docs/en/compute/AWS-EMR.md [hunk 2]
```diff
@@ -152,16 +152,85 @@ Tuning of Alluxio properties can be done in a few different locations.
...
+alluxio-emr.sh is a script which can be used to bootstrap an AWS EMR cluster
+with Alluxio. It can download and install Alluxio as well as add properties
+specified as arguments to the script.
+  
+By default, if the environment this script executes in does not already contain
+an Alluxio install at /opt/alluxio then it will download, untar, and configure
+the environment at /opt/alluxio. If an install already exists at /opt/alluxio,
+nothing will be installed over it, even if -d is specified.
+  
+If a different Alluxio version is desired, see the -d option.
```
Comment (ZacBlanco) [inline docs/en/compute/AWS-EMR.md]: maybe instead say if a "specific Alluxio tarball" instead?

File: docs/en/compute/AWS-EMR.md [hunk 3]
```diff
@@ -152,16 +152,85 @@ Tuning of Alluxio properties can be done in a few different locations.
...
+
 ### Alluxio Service
 
-Any server-side configuration changes must be made in the `alluxio-emr.sh` bootstrap script.
-In the section for generating the `alluxio-site.properties`, add a line with the configuration
-needed to append to the bottom of the file.
-Options can also be passed as the 3rd argument to the bootstrap script with a ';' delimiter.
+Making configuration changes to the Alluxio Service can be done in a few different ways via the
+bootstrap script.
+The [-p] flag allows users to pass in a set of delimited key-value properties to be set on all of
```
Comment (ZacBlanco) [inline docs/en/compute/AWS-EMR.md]: backtick around `[-p]`

File: docs/en/compute/AWS-EMR.md [hunk 4]
```diff
@@ -152,16 +152,85 @@ Tuning of Alluxio properties can be done in a few different locations.
...
+the Alluxio nodes. An alternative would be to pass in a custom file using the [-f] flag named
```
Comment (ZacBlanco) [inline docs/en/compute/AWS-EMR.md]: line break on sentence

Comment (ZacBlanco) [inline docs/en/compute/AWS-EMR.md]:
```suggestion
the Alluxio nodes.
An alternative would be to pass in a custom file using the `[-f]` flag named
```

File: docs/en/compute/AWS-EMR.md [hunk 5]
```diff
@@ -152,16 +152,85 @@ Tuning of Alluxio properties can be done in a few different locations.
...
+The bootstrap also allows users to install previous versions of Alluxio (>=2.0) by specifying
+a download URL (HTTP or S3 only).
 
 ### Alluxio Client
 
 Generic client-side properties can also be edited via the bootstrap script as mentioned above.
 This is mostly for the native client (CLI).
 Property changes for a specific service like Presto/Hive should be done in the respective
-configuration file i.e. `core-site.xml`, `hive.catalog`.
+configuration file i.e. `core-site.xml`, `hive.catalog` via the EMR JSON entry-point.
```
Comment (ZacBlanco) [inline docs/en/compute/AWS-EMR.md]: is entry-point the right word here?

File: docs/en/compute/AWS-EMR.md (hunk 3 above)
Comment (ZacBlanco) [inline docs/en/compute/AWS-EMR.md]:
```suggestion
The `[-p]` flag allows users to pass in a set of delimited key-value properties to be set on all of
```

File: docs/en/compute/AWS-EMR.md (hunk 5 above)
Comment (ns1123) [reply docs/en/compute/AWS-EMR.md]: I think it's fine. Do you have a better suggestion?

File: docs/en/compute/AWS-EMR.md (hunk 2 above)
Comment (ns1123) [reply docs/en/compute/AWS-EMR.md]: This should be modified in the bootstrap README. I only pasted the copy into the docs so users wouldn't have to download and view it themselves.

File: docs/en/compute/AWS-EMR.md (hunk 1 above)
Comment (ns1123) [reply docs/en/compute/AWS-EMR.md]: @Xenorith @tieujason330 Please comment on this? It's an AWS artifact.

File: docs/en/compute/AWS-EMR.md (hunk 2 above)
Comment (ZacBlanco) [reply docs/en/compute/AWS-EMR.md]: agreed. edit here and update?

Comment (ns1123) [reply docs/en/compute/AWS-EMR.md]: done

File: docs/en/compute/AWS-EMR.md (hunk 1 above)
Comment (Xenorith) [reply docs/en/compute/AWS-EMR.md]: lgtm

**Comment by ZacBlanco** [comment]
alluxio-bot, merge this please

**Comment by ZacBlanco** [comment]
alluxio-bot, cherry-pick this to branch-2.1 please

**Comment by ZacBlanco** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please

**Pull request merged**